File: URI_Online_Judge_1070.c
#include <stdio.h>
 
int main() {

	int i, X,odd;
	
	scanf("%d",&X);
	
		for(i = X; i < X+12 ; i = i+2){
			
			if(X%2==0){
				
				odd = i + 1;
				printf("%d\n",odd);
				
			}else{
				
				odd = i;
				printf("%d\n",odd);
			}
	}
		
    return 0;
}

File: URI_Online_Judge_1070.java
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		int X,odd;
		Scanner input = new Scanner(System.in);
			
		X = input.nextInt();
		
		for(int i = X; i < X+12 ; i = i+2){
			
			if(X%2==0){
				
				odd = i + 1;
				System.out.println(odd);
				
			}else{
				
				odd = i;
				System.out.println(odd);
			}
		}
	}
}